### PR TITLE
fixes #3337 usage of old v1 table names

### LINKED ISSUE #3337: threadGetNew() uses not existing table
Query uses forum_t table from version 1

```
	function threadGetNew($count = 50, $unread = true, $uid = USERID)
	{
		$sql = e107::getDb();
		$viewed = '';
		if($unread)
		{
			$viewed = implode(',', $this->threadGetUserViewed($uid));
			if($viewed != '')
			{
				$viewed = ' AND p.post_forum NOT IN ('.$viewed.')';
			}
		}
		$qry = "
		SELECT ft.*, fp.thread_name as post_subject, fp.thread_total_replies as replies, u.user_id, u.user_name, f.forum_class
		FROM #forum_t AS ft
		LEFT JOIN #forum_thread as fp ON fp.thread_id = ft.thread_parent
		LEFT JOIN #user as u ON u.user_id = SUBSTRING_INDEX(ft.thread_user,'.',1)
		LEFT JOIN #forum as f ON f.forum_id = ft.thread_forum_id
		WHERE ft.thread_datestamp > ".USERLV. "
		AND f.forum_class IN (".USERCLASS_LIST.")
		{$viewed}
		ORDER BY ft.thread_datestamp DESC LIMIT 0, ".intval($count);
		$qry = "
		SELECT t.*, u.user_name FROM `#forum_thread` AS t
		LEFT JOIN `#user` AS u ON u.user_id = t.thread_lastuser
		WHERE t.thread_lastpost > ".USERLV. "
		{$viewed}
		ORDER BY t.thread_lastpost DESC LIMIT 0, ".(int)$count;
		if($sql->gen($qry))
		{
			$ret = $sql->db_getList();
		}
		return $ret;
	}
```

### PR DESCRIPTION
**Pull request by SimSync**
- fixes usage of old v1 table names
- new route /forum/?new to display new posts

### CODE REVIEW

File: e107_plugins/forum/e_url.php [hunk 1]
```diff
@@ -50,6 +50,12 @@ function config()
 			'legacy'        => '{e_PLUGIN}forum/forum.php?f=mfar&id={forum_id}'
 		);
 
+		$config['new']  = array(
+			'regex'			=> '^forum/\?new?',
+			'sef'           => 'forum/?new',
+			'redirect'      => '{e_PLUGIN}forum/forum.php?new'
+		);
+
```
[the remark this answers is not in the document]
Comment (CaMer0n) [reply e107_plugins/forum/e_url.php]: Then maybe just make it a proper sef URL.  'forum/new'

[the remark this answers is not in the document]
Comment (SimSync) [reply e107_plugins/forum/e_url.php]: hmm... hopefully nobody creates a forum with name/sef "new" than... 😆

**Comment by SimSync** [comment]
Updated the route to `/forum/new` instead of `/forum/?new`

File: e107_plugins/forum/e_url.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (CaMer0n) [reply e107_plugins/forum/e_url.php]: hmm, that's a good point. :thinking:

**Comment by SimSync** [comment]
@CaMer0n Did you see my new route proposal?

**Comment by CaMer0n** [comment]
Looks good! Thanks :+1:

**Pull request merged**